packages/perseus/src/util/parse-perseus-json/perseus-parsers/radio-widget.ts

```diff
@@ -19,7 +20,7 @@ export const parseRadioWidget: Parser<RadioWidget> = parseWidget(
     object({
         choices: array(
             object({
-                content: string,
+                content: defaulted(string, () => ""),
```

Comment:
The danger with that approach is that if the default value is an object or array, the same instance will be reused for every default. E.g.

```ts
defaulted(array(someParser), [])
```

If some other code later mutates the array, it might affect unrelated code that happens to be holding a reference to the same value. This kind of "aliasing" bug has bitten me many times in my career, and it's always a nightmare.

I believe we have a pretty good culture of avoiding unnecessary mutation, but aliasing bugs are so bad that I feel it's better to be safe than sorry.

Reply:
Fair point. Let's leave it as a function then.